spaceship.zsh

```diff
@@ -411,11 +411,9 @@ spaceship_git_branch() {
 
   _is_git || return
 
-  local ref
-  ref=$(command git symbolic-ref --quiet HEAD 2> /dev/null)
-  local ret=$?
   if [[ $ret != 0 ]]; then
     [[ $ret == 128 ]] && return
+  local ref=$(command git symbolic-ref --quiet HEAD 2> /dev/null) ret=$?
```

Comment:
This change confuses me. Wouldn't `$ret` empty above? Why then you execute the same command on the next line?